roda-ui/roda-wui/src/main/java/org/roda/wui/api/controllers/RODAInstance.java

```diff
@@ -462,4 +470,36 @@ public static String removeSyncBundle(String bundleName, User user, String bundl
     return RODAInstanceHelper.removeSyncBundle(bundleName, bundleDirectory);
 
   }
+
+  public static Long retrieveUpdates(User user, String instanceIdentifier)
```

Comment:
Change method name to "retrieveCentralInstanceUpdates"

Reply:
done